HealthMonitoring.Forwarders.Influxdb/HealthMonitoring.Forwarders.Influxdb.csproj

```diff
@@ -0,0 +1,60 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
i was using it for coordinator testing. Should i remove it or what ? we'll need this on next stage of this task.